infrastructure/tooling/src/smoketest/checks/succeed.js

```diff
@@ -0,0 +1,46 @@
+const taskcluster = require('taskcluster-client');
+
+exports.tasks = [];
+exports.tasks.push({
+  title: 'Built-in/succeed task check',
+  requires: [],
+  provides: [
+    'built-in/succeed',
+  ],
+  run: async (requirements, utils) => {
+    let task = {
+      provisionerId: 'built-in',
+      workerType: 'succeed',
+      created: (new Date()).toJSON(),
+      deadline: taskcluster.fromNowJSON('2 minutes'),
+      metadata: {
+        name: "Smoketest built-in/succeed",
+        description: "built-in/succeed task created during smoketest",
+        owner: "smoketest@taskcluster.net",
+        source: "https://taskcluster.net",
+      },
+      payload: {},
+    };
+    let taskId = taskcluster.slugid();
+    let queue = new taskcluster.Queue({
```

Comment:
you can use `taskcluster.Queue(taskcluster.fromEnvVars())` as a shorter way to write the same thing.